server/models/events_persister.go

```diff
@@ -154,13 +154,13 @@ func (e *EventsPersister) PersistEvent(event *events.Event) error {
 	return nil
 }
 
-func (e *EventsPersister) getCountBySeverity(userID uuid.UUID, eventStatus events.EventStatus) ([]*CountBySeverityLevel, error) {
+func (e *EventsPersister) getCountBySeverity(userID, sysID uuid.UUID, eventStatus events.EventStatus) ([]*CountBySeverityLevel, error) {
 	if eventStatus == "" {
 		eventStatus = events.Unread
 	}
 
 	eventsBySeverity := []*CountBySeverityLevel{}
-	err := e.DB.Model(&events.Event{}).Select("severity, count(severity) as count").Where("status = ? and user_id = ?", eventStatus, userID).Group("severity").Find(&eventsBySeverity).Error
+	err := e.DB.Model(&events.Event{}).Select("severity, count(severity) as count").Where("status = ? AND (user_id = ? OR user_id = ?)", eventStatus, userID, sysID).Group("severity").Find(&eventsBySeverity).Error
```

Comment:
@jerensl  why is or agains user_id and user_id . i think it should be system_id .